p2p.go

```diff
@@ -100,6 +103,8 @@ var (
 		RateLimit:                DefaultRatelimitConfig,
 		PrivateNetworkPSK:        "",
 		ProtocolID:               "/iotex",
+		PeersRefreshInterval:     30 * time.Minute,
+		GroupID:                  "iotex",
```

Comment:
we can use `iotex + strconv.Itoa(int(chainID)` similar to ProtocolID(), then don't need this value in config